canonical-web-and-design/jaas-dashboard
# Add top padding to panels in model details

## Done

Add top padding to panels in model details

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check that the model details panel have a top padding to align content with table headers


## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1595

<img width="1241" alt="Screenshot 2021-01-20 at 12 59 43" src="https://user-images.githubusercontent.com/505570/105178257-b264db00-5b1f-11eb-9729-5c643d1ec911.png">
